kedro/pipeline/node.py

```diff
@@ -197,20 +197,14 @@ def __call__(self, **kwargs) -> Dict[str, Any]:
         return self.run(inputs=kwargs)
 
     @property
-    def _func_name(self):
-        if hasattr(self._func, "__name__"):
-            return self._func.__name__
-
-        name = repr(self._func)
-        if "functools.partial" in name:
+    def _func_name(self) -> str:
+        name = _get_readable_func_name(self._func)
+        if name == "<partial>":
             warn(
-                "The node producing outputs `{}` is made from a `partial` function. "
+                f"The node producing outputs `{self.outputs}` is made from a `partial` function. "
                 "Partial functions do not have a `__name__` attribute: consider using "
```

Comment:
We try follow a convention where all f-string parts are prefixed with `f` even if there are no variables inside. It just for the consistency with how PyCharm, for example, breaks down such strings.
```suggestion
                f"Partial functions do not have a `__name__` attribute: consider using "
```